issue_release_status.go

```diff
@@ -88,7 +88,17 @@ func (handler *PRCreateHandler) Handle(ctx context.Context, eventType, deliveryI
 	prNum := event.GetNumber()
 	installationID := githubapp.GetInstallationIDFromEvent(&event)
 
-	ctx, logger := githubapp.PreparePRContext(ctx, installationID, repository, prNum)
+	logctx := zerolog.Ctx(ctx).With()
+	logctx = logctx.Int64("github_installation_id", installationID)
+	logctx = logctx.Str("github_repository_owner", repository.GetOwner().GetLogin())
+	logctx = logctx.Str("github_repository_name", repository.GetName())
+	logctx = logctx.Int("github_pr_num", prNum)
```

Comment:
This could be chained as well, right?

```suggestion
	logctx := zerolog.Ctx(ctx).With().
		Int64("github_installation_id", installationID).
		Str("github_repository_owner", repository.GetOwner().GetLogin()).
		Str("github_repository_name", repository.GetName()).
		Int("github_pr_num", prNum)
```

Reply:
Yes it can